department-of-veterans-affairs/va.gov-cms
# VACMS-4766: Improve health service selector.

## Description

See #4766. 

## Testing done

Manual.


## QA steps

1. Edit taxonomy term 152 (Minority Veteran care), and note that Type of Care is "Social programs as services"
2. Add "Counseling services" to the Vet Center Type of Care field
3. Go to /node/add/vet_center_facility_health_servi 
4. You should see Minority Veteran Care listed as "Minority Veteran care - Counseling services"
5. Go to /node/add/regional_health_care_service_des
6. You should see Minority Veteran Care listed as "Minority Veteran care - Social programs as services"


### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
